### ISSUE TITLE
When pretrained, what was the accuracy of the masked-LM task?

### ISSUE THREAD

**Issue by bgshin** [google-research/bert#427]
When pretrained, what was the accuracy of the masked-LM task?

**Comment by batteryhp** [comment]
+1, I have the same question.

**Comment by qiunian711** [comment]
+1, I have the same question.

**Comment by bgshin** [comment]
May we can reference this? (I found this in the README.md)
So the accuracy is 0.985479?

***** Eval results *****
  global_step = 20
  loss = 0.0979674
  masked_lm_accuracy = 0.985479
  masked_lm_loss = 0.0979328
  next_sentence_accuracy = 1.0
  next_sentence_loss = 3.45724e-05

**Comment by johnmoberg** [comment]
That refers to training on ~30 sentences: "Note that since our sample_text.txt file is very small, this example training will overfit that data in only a few steps and produce unrealistically high accuracy numbers."
